feat(tables): adding border prop (#784)
cengage/react-magma: website/react-magma-docs/src/pages/api/table.mdx

  ];

  return (
    <Table>
      <TableHead>
        <TableRow>
          <TableHeaderCell>Column</TableHeaderCell>
          <TableHeaderCell>Column</TableHeaderCell>
          <TableHeaderCell>Column</TableHeaderCell>
          <TableHeaderCell>Column</TableHeaderCell>
        </TableRow>
      </TableHead>
      <TableBody>
        {rows.map((row, i) => (
          <TableRow key={`row${i}`}>
            {row.map((cell, j) => (
              <TableCell key={`cell${i}_${j}`}>{cell}</TableCell>
            ))}
          </TableRow>
        ))}
      </TableBody>
    </Table>
  );
}
```

## Table With Border

> The `hasTableBorder` prop is a boolean which gives the table a border. By default the table border is off.

Reviewer: ```suggestion
The `hasOuterBorder ` prop is a boolean which gives the table an outer border. By default the border is off.
```

```tsx
import React from 'react';
import {
  Table,
  TableHead,
  TableRow,
  TableHeaderCell,
  TableBody,
  TableCell,
} from 'react-magma-dom';

export function Example() {
  const rows = [
    [
      'Lorem ipsum dolor sit amet consectetur',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
    [
      'Lorem ipsum dolor sit amet',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
    [
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
    [
      'Lorem ipsum dolor sit amet consectetur',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
    [
      'Lorem ipsum dolor sit amet',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
    [
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum dolor',
      'Lorem ipsum',
    ],
  ];

  return (

>     <Table hasTableBorder>

Reviewer: ```suggestion
    <Table hasOuterBorder>
```

  TableHeaderCell,
  TableBody,
  TableCell,
  TableSortDirection,
  magma,
} from 'react-magma-dom';

export function Example() {
  return (
    <Card isInverse>
      <Table
        isInverse
        hasZebraStripes
        hasVerticalBorders
        hasHoverStyles

>         hasTableBorder

Reviewer: ```suggestion
        hasOuterBorder
```
